### DISCUSSION TITLE
3rd party AV

### DISCUSSION THREAD

**Discussion by susan298** [amidaware/tacticalrmm#991] [Q&A]
Hi,
I know the documentation says no 3rd party AV, but is there a way to get it to work with McAfee. that's the only thing stopping me from using this. McAfee keeps marking it as trojan even when I exclude it.

**Comment by dinger1986** [comment]
code signing might help, theres also loads of examples of exclusions in the docs, I would imagine you havent got them all in, https://wh1te909.github.io/tacticalrmm/av/. Maybe need to exclude the folders and processes. Also you can try powershell. Tbf with the scripts for managing defender in tactical its pretty powerful.

**Comment by silversword411** [comment]
If you get it working, and get screenshots of all the places and share I'll add them to the docs! :)

**Comment by silversword411** [comment]
> Hi, I know the documentation says no 3rd party AV

...we _recommend_ no 3rd party because they're a pain and slow your computer down more than necessary. But some people need it for reporting/compliance, so you'll need to add exclusions manually. Can always try code signing which usually helps.
